### DISCUSSION TITLE
大佬看看这个问题

### DISCUSSION THREAD

**Discussion by ghost** [eycorsican/leaf#66] [Q&A]
您好，大佬。我单纯的trojan协议不加websocket，就trojan+tls。在iPhone12上上传instagram的视频无法成功。
配置如下

[General]
dns-server = 114.114.114.114, 223.5.5.5
always-real-ip = tracker, apple.com
tun = replace-tun
loglevel = trace

[Proxy]
Direct = direct
Trojan = trojan, us6r679.wwap1.com, 443, password=jTOU3V, tls=true
[Rule]
EXTERNAL, site:cn, Direct
EXTERNAL, mmdb:cn, Direct
FINAL, Trojan

不知您是否在手机真机上遇到？

https://user-images.githubusercontent.com/46473120/108972626-7a245f80-76be-11eb-8e7e-e3470bb7c0d8.mp4

请看视频,就像这样子,卡在那里再也不会请进

**Comment by eycorsican** [comment]
你没提过任何服务端实现相关的信息，建议你测试用 leaf 作服务端，也把所有相关日志发上来

**Reply by ghost** [reply]
> 你没提过任何服务端实现相关的信息，建议你测试用 leaf 作服务端，也把所有相关日志发上来

后端使用trojan-go搭建 v0.8.0
[trojan-go.log](https://github.com/eycorsican/leaf/files/6042588/trojan-go.log)
 trojan-go的日志

leaf日志
[2021-02-25.log](https://github.com/eycorsican/leaf/files/6042714/2021-02-25.log)

**Reply by ghost** [reply]
> 你没提过任何服务端实现相关的信息，建议你测试用 leaf 作服务端，也把所有相关日志发上来

大佬能告诉下我思路吗？

**Comment by eycorsican** [comment]
用现在master的代码还有这问题吗？

**Reply by ghost** [reply]
> 用现在master的代码还有这问题吗？

还是有的

**Reply by ghost** [reply]
> 用现在master的代码还有这问题吗？

[2021-03-01.log](https://github.com/eycorsican/leaf/files/6062166/2021-03-01.log)

**Reply by eycorsican** [reply]
我说让你用 leaf 作服务端呢？

**Reply by ghost** [reply]
> 我说让你用 leaf 作服务端呢？

我改下试试，我以为现在用trojan-go也可以了呢

**Comment by giftedunicorn** [comment]
解决了吗？

**Comment by ghost** [comment]
> 解决了吗？

没有

**Comment by giftedunicorn** [comment]
我发现连接trojan之后，用speedtest测速，download正常，但是upload就零点几

**Reply by ghost** [reply]
你问下作者吧

**Reply by ixiao** [reply]
我们这边后台用的Trojan-Go，移动端用leaf，但是速度不是很快，rust有迁移成本 我们也在考虑
